### DISCUSSION TITLE
[Feature] Add "View all places"

### DISCUSSION THREAD

**Discussion by aisbergde** [immich-app/immich#6273] [Feature Request]
### The feature

I had hoped that the button for "All Places" would be implemented with 1.92.0
Presumably a separate feature request is necessary.

![image](https://github.com/immich-app/immich/assets/9020187/a9f66d3d-b447-4bc1-8972-566b61d6564a)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Closing as a duplicate of #7070
